pkg/config/config_template.yaml

```diff
@@ -479,6 +479,9 @@ api_key:
 # URL where the ECS agent can be found. Standard cases will be autodetected.
 # ecs_agent_url: http://localhost:51678
 #
+# Name of the ecs-agent docker container, for autodetection.
```

Comment:
This can be confusing if someone reads through the file and doesn't realize there's a default value. Let's state that it only needs to be set if they have a custom one and that the default is `ecs-agent`.